code/api/src/modules/product/model.js

```diff
@@ -22,4 +22,6 @@ module.exports = function(sequelize, DataTypes) {
       type: DataTypes.TEXT
     }
   })
-}
\ No newline at end of file
+
+  return Product
+}
```

Comment:
Not 100% confident this change is required, but mirrored the way User model was structured for using before/after blocks.